### DISCUSSION TITLE
How to add a new currency ?

### DISCUSSION THREAD

**Discussion by rohitsingal1** [inventree/InvenTree#4863] [Q&A] [answered]
I would like to INR(Indian Rupees) as a currency in  Inventree

**Comment by SchrodingersGat** [comment, marked as answer]
@rohitsingal1 you need to adjust your configuration file to support more currency types:

https://docs.inventree.org/en/latest/start/config/#supported-currencies

**Reply by rohitsingal1** [reply]
Thanks @SchrodingersGat

**Comment by rohitsingal1** [comment]
Added the following in the configuration file and restarted Inventree. Still cannot see the new currency.

```

> # Base currency code (or use env var INVENTREE_BASE_CURRENCY)
> base_currency: INR
> 
> # Add new user on first startup
> #admin_user: admin
> #admin_email: info@example.com
> #admin_password: inventree
> 
> # List of currencies supported by default. Add other currencies here to allow use in InvenTree
> currencies:
>   - AUD
>   - CAD
>   - CNY
>   - EUR
>   - GBP
>   - JPY
>   - NZD
>   - USD
>   - INR
> 

```

**Comment by rohitsingal1** [comment]
Done. Was able to get the currency after refreshing the currency screen.

**Reply by SchrodingersGat** [reply]
Glad you got it sorted :)

**Reply by rohitsingal1** [reply]
It's a good feature. A suggestion if we could change the currency symbol as well it would be helpful. INR has a rupee symbol.

**Reply by matmair** [reply]
@rohitsingal1 We are using a library for rendering so changing this behavior will take additional core code. Most currencies are shown with their 3 letter abbreviation to ensure they are rendered correctly/consistently in as many fonts as possible.

**Reply by rohitsingal1** [reply]
@matmair got this one sorted. Thanks
